### PR TITLE
Minify css

### PR DESCRIPTION
**Pull request by oetherington**
Yesterday we noticed that our generated CSS isn't getting minified - this fixes that.

* Without minification: 590.1kB (82.2kB gzipped)
* With minification: 474.7kB (75.3kB gzipped)

So, about a 20% improvement without gzip, or 7% with.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203277324062083) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by jpaddison3** [comment]
I dunno if 7% of a small file is worth the pain of not being able to inspect the prod site and see the classes. Sigh. I guess it's probably worth it? Probably worth it. QQ: Does it minify if I run `yarn ea-start-prod-db`? I assume no.

File: packages/lesswrong/server/styleGeneration.tsx [hunk 1]
```diff
@@ -51,8 +52,9 @@ const generateMergedStylesheet = (themeOptions: ThemeOptions): Buffer => {
     jssStylesheet,
     ...theme.rawCSS,
   ].join("\n");
-  
-  return Buffer.from(mergedCSS, "utf8");
+
+  const minifiedCSS = minify(mergedCSS).css;
+  return Buffer.from(minifiedCSS, "utf8");
```
Comment (jpaddison3) [inline packages/lesswrong/server/styleGeneration.tsx:57]: Wait, this doesn't seem to have any prod checking. We **definitely** don't want to do this on dev.

**Comment by oetherington** [comment]
@jpaddison3 This PR doesn't do any class name mangling - it just removes the whitespace and comments. It's not obvious to me that that's bad on dev, since it's already much more annoying to inspect the actual style sheet than the values in the dev console. Opinions?

**Comment by jpaddison3** [comment]
Ah! Mucho better.

Prod-gating pros vs cons:

Pros:
 * Slightly nicer in case you have to look at raw css in debugging
 * No need to save bandwidth

Cons:
 * Dev-prod parity

I think dev-prod parity wins.

:shipit:

**Review by jpaddison3 — APPROVED** [review]

**Pull request merged**